resources/assets/js/label-trees/components/labelTreeLabel.vue

```diff
@@ -12,11 +12,13 @@
                 <span v-text="label.name" @click.stop="toggleSelect" @mouseenter="dontHover"></span>
             </span>
             <span class="label-tree-label__buttons">
-                <button v-if="showFavourites" type="button" class="label-tree-label__favourite" :class="favouriteClass" @click.stop="toggleFavourite" :title="favouriteTitle">
-                    <span v-if="showFavouriteShortcuts" @click.stop="!toggleFavourite" >
-                        <span class="fa fa-keyboard" aria-hidden="true" title=""></span>
-                        <span v-text="addOnePosition()"></span>
+                <span class="label-tree-label_position">
+                    <span v-if="showFavouriteShortcuts" class="text-muted">
+                            <span class="fa fa-keyboard" aria-hidden="true" title=""></span>
+                            <span :text="actualPosition">{{ actualPosition }}</span>
```

Comment:
My fault, it should have been `v-text` instead of `:text`. But this is also fine as it does the same thing:
```suggestion
                            <span>{{ actualPosition }}</span>
```

I avoid the `{{}}` and use `v-text` instead in the Blade templates of Laravel. However, here in the Vue template it's fine.